source/sgp_mode/Instructions.h

```diff
@@ -96,6 +96,25 @@ INST(IO, {
   *a = next;
   state.input_buf.push(next);
 });
+
+INST(Reuptake, {
+  uint32_t next;
+  float score = state.world->GetTaskSet().CheckTasks(state, *a);
+  if (score != 0.0) {
+    state.host->AddPoints(pow(2, score));
+    if (!state.host->IsHost()) {
```

Comment:
We should make a note to fix the name of state.host so that it isn't 'host' if it might not always be a host